runtime-decompiler/src/main/java/org/jrd/frontend/frame/main/BytecodeDecompilerView.java

```diff
@@ -36,6 +36,7 @@
 import java.awt.event.KeyEvent;
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
+import java.awt.event.MouseMotionListener;
```

Comment:
I need to tray this :D

Reply:
How is it supposed to work?
I was not able to seelct more then one.
Also select all or ctrl/shift selects do nto work

idea is good, implementation to not brinf wha tit promisses.


btw - the copy indiivudal names/loader/location is great, but copy all three together is missing.
btw2 - our regexe based selector is now not enough. There shoudl be three (maybe hidden behind osme ... menu.. maybe popupmenu with three checkbox on the button where was originally copy class names) checkboxes, which shold point what parts of info to match,.

Reply:
hmm. the drag really works. weird.

Reply:
maybe put and disabled item into the copy submenu, explaining how to select multiple?

Reply:
> How is it supposed to work? I was not able to seelct more then one. Also select all or ctrl/shift selects do nto work

Were you not pulled?

> hmm. the drag really works. weird.

I know, I spent multiple hours deep in Swing shenanigans for this feature.

> maybe put and disabled item into the copy submenu

Good idea.

> Also select all or ctrl/shift selects do nto work

Yeah, you'll notice it's a MouseAdapter and not a KeyAdapter. I was not able to come up with a scheme that would prevent the user from exploiting multi selection being allowed due to ctrl/shift being pressed, but using the mouse in unison to select. Chaos ensued.